SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -482,11 +500,100 @@ int64 USpatialActorChannel::ReplicateActor()
 	return (bWroteSomethingImportant) ? 1 : 0;	// TODO: return number of bits written (UNR-664)
 }
 
-bool USpatialActorChannel::ReplicateSubobject(UObject* Object, const FClassInfo& Info, const FReplicationFlags& RepFlags)
+void USpatialActorChannel::DynamicallyAttachSubobject(UObject* Object)
+{
+	// Find out if this is a dynamic subobject or a subobject that is already attached but is now replicated
+	FUnrealObjectRef ObjectRef = NetDriver->PackageMap->GetUnrealObjectRefFromObject(Object);
+
+	const FClassInfo* Info = nullptr;
+
+	// Subobject that's a part of the CDO by default does not need to be created.
+	if (ObjectRef.IsValid())
+	{
+		Info = &NetDriver->ClassInfoManager->GetOrCreateClassInfoByObject(Object);
+	}
+	else
+	{
+		Info = TryResolveNewDynamicSubobjectAndGetClassInfo(Object);
+
+		if (Info == nullptr)
+		{
+			return;
```

Comment:
Add a comment that this is a failure, but there's already a log inside `TryResolveNewDynamicSubobjectAndGetClassInfo`.